### DISCUSSION TITLE
Does not work in subdirectories

### DISCUSSION THREAD

**Discussion by KUNKUN2121** [filamentphp/filament#14843] [Help]
### Package

Other

### Package Version

3.2

### How can we help you?

If you install Laravel in a subdirectory,
you will not be able to access the Laravel Filament.

`APP_URL=https://dev.AAA.jp/laravvel/ `

When you access `https://dev.AAA.com/laravel/admin/`, 
you will be redirected to `https://dev.AAA.com/admin/login/`

If I change the laravel filament path to panel/admin I get a 404.
The route in web.php works.
Is there any solution?

The APP_URL in the .env file has been changed.
I am also using nginx reverse proxy.

**Comment by leandrocfe** [comment]
Why not subdomain instead of subdirectory? I think subdomain is a better approach to do this.
